LabeledIconButton/LabeledIconButton.js

```diff
@@ -76,11 +97,29 @@ const LabeledIconButtonBase = kind({
 		publicClassNames: true
 	},
 
-	render: ({css, icon, pressed, selected, spriteCount, ...rest}) => {
+	render: ({
+		backgroundOpacity,
+		css,
+		icon,
+		iconComponent,
+		highlighted,
+		pressed,
+		selected,
+		spriteCount,
+		...rest
+	}) => {
 		return UiLabeledIcon.inline({
 			...rest,
 			icon: (
-				<Button pressed={pressed} selected={selected} spriteCount={spriteCount}>{icon}</Button>
+				<Button
+					backgroundOpacity={backgroundOpacity}
+					icon={icon}
+					iconComponent={iconComponent}
+					highlighted={highlighted}
+					pressed={pressed}
+					selected={selected}
+					spriteCount={spriteCount}
```

Comment:
Is this prop gone now? I'm seeing warnings about it and can't find references any longer.

Reply:
This prop needs to be restored onto Button and sent along to the inner Icon component. I think we may need to update enact/ui to properly add this.

Reply:
How do you want to proceed for this PR?

Reply:
I will spend a few minutes and see if I can make it joyful again.

Reply:
Let's ignore this for now and address it later. I'm not having any luck fixing it efficiently.